frontend/src/components/MenuBar/MenuBar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+//import { TouchableOpacity, Text } from "react-native";
+//import { createNativeStackNavigator } from "@react-navigation/native-stack";
+//import { NavigationContainer } from "@react-navigation/native";
+//import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import { useNavigation } from "@react-navigation/native";
+import { View, Text, TouchableHighlight } from "react-native";
+import { styleVariables } from "../../styles/globalStyleVariables";
+import Button from "../Button/Button";
+
+export default function MenuBar() {
+  const navigation = useNavigation();
+  return (
+    <View style={styles.container}>
+      <Button title="Clear all" />
```

Comment:
As mentioned, this was not a part of this ticket but great to have them here, just don't worry about the styling.